src/helm/benchmark/run_specs/lite_run_specs.py

```diff
@@ -255,6 +255,26 @@ def get_legalbench_spec(subset: str) -> RunSpec:
     )
 
 
+@run_spec_function("med_qa")
+def get_med_qa_spec() -> RunSpec:
+    scenario_spec = ScenarioSpec(class_name="helm.benchmark.scenarios.med_qa_scenario.MedQAScenario", args={})
+
+    adapter_spec = get_multiple_choice_adapter_spec(
+        method=ADAPT_MULTIPLE_CHOICE_JOINT,
+        instructions="The following are multiple choice questions (with answers) about medicine.",
```

Comment:
@MiguelAFH, in the prompt, we say "multiple choice questions (with answers)".  Is this directly from HELM?

I was wondering if it was worth standardizing to the Medbullets style prompt - https://crfm.stanford.edu/helm/medhelm/latest/#/runs/medbullets:num_output_tokens=4000,model=openai_o3-mini-2025-01-31,model_deployment=stanfordhealthcare_o3-mini-2025-01-31